src/rvpt/timer.cpp

```diff
@@ -0,0 +1,45 @@
+//
+// Created by legend on 5/27/20.
+//
+
+#include <algorithm>
+#include "timer.h"
```

Comment:
timer.h should be included first. (for .cpp files, the header include order is the same as header files, just with the .cpp's header files first.